foreshadow/cleaners/internals/datetimes.py

```diff
@@ -16,15 +16,14 @@ def _split_to_new_cols(t):
 
     """
     delimiters = "[-/]"
-    regex = r"^.*(([\d]{4})%s([\d]{2})%s([\d]{2})).*$" % (
-        delimiters,
-        delimiters,
+    regex = r"^.*(([\d]{{4}}){delim}([\d]{{2}}){delim}([\d]{{2}})).*$".format(
+        delim=delimiters
     )
     text = str(t)
     res = re.search(regex, text)
     if res is not None:
         res = sum([len(range(reg[0], reg[1])) for reg in res.regs[1:2]])
-        texts = [re.sub(regex, r"\%d" % i, text) for i in range(2, 5)]
+        texts = [re.sub(regex, r"\{}".format(i), text) for i in range(2, 5)]
```

Comment:
you can use this: {:d!} if you want it to force it to be an int (not a suggested change here as `i` will always be an int.

Reply:
Missed refactoring this, will fix